packages/venia-concept/src/components/Gallery/item.js

```diff
@@ -112,6 +113,9 @@ class GalleryItem extends Component {
     /**
      * Product images are currently broken and pending a fix from the `graphql-ce` project
      * https://github.com/magento/graphql-ce/issues/88
+     *
+     * When using sample data, which symlinks to bypass cache,
+     * you can simple prepend /media/catalog/product/, which we'll do from .env.
```

Comment:
Good to know. 👍

Reply:
Will this leave broken images for those not using the sample data? Anton said there is a convention we can use to get _any_ product photo with the URL we get back from GraphQL. It will be the full size (for now), but we wouldn't have to hack it

Reply:
Yeah it would. And that would be awesome--we can hit it next sprint.